src/hotspot/share/classfile/verificationType.hpp

```diff
@@ -242,11 +242,11 @@ class VerificationType {
   bool is_double_array() const { return is_x_array(JVM_SIGNATURE_DOUBLE); }
   bool is_object_array() const { return is_x_array(JVM_SIGNATURE_CLASS); }
   bool is_array_array() const { return is_x_array(JVM_SIGNATURE_ARRAY); }
-  bool is_value_array() const { return is_x_array(JVM_SIGNATURE_VALUETYPE); }
+  bool is_inline_array() const { return is_x_array(JVM_SIGNATURE_INLINE_TYPE); }
```

Comment:
The term "inline" alone can be misleading (is it about the element being an inline type, is it about the elements of the array being inlined in the array, is it about the array being inlined in something else, etc).
Here, the check is about the kind of elements stored in the array, so I would suggest "is_inline_type_array()".